pug/mixins/mixins.pug

```diff
@@ -73,3 +73,8 @@ mixin breadcrumb(items, last)
                 li.breadcrumb__item
                     span.breadcrumb__link-current= last
             else
+
+mixin input-group(lable,forlable) 
+    div.form-group 
+        label.text-require(for="forlable")=lable
+        input.input(type="text")
```

Comment:
```suggestion
mixin input-group(label, type = 'text') 
    div.form-group 
        label.text-require()=label
            input.input(type=type)
```
for thì phải đi kèm với ID, để khi người dùng nhấn vào cái label, thì nó sẽ tự động focus vào input luôn.
Còn k muốn để ID thì bọc input bên trong label là được. Ngoài ra thì bổ sung thêm type, hiện tại Email với Password đang để type là text hết kìa